src/components/Template.tsx

```diff
@@ -50,16 +61,33 @@ class Template extends React.Component<Props, State> {
   };
 
   async componentDidMount() {
-    this.props.getWallets();
+    this.props.bootstrap();
     this.props.startDeviceScan();
-    this.props.getSettings();
   }
 
   componentWillUnmount() {
     this.props.stopDeviceScan();
   }
+
   render() {
-    const { wallets, activeWallet } = this.props;
+
+    const { wallets, activeWallet, ready, hasValidSettings, hasWallets } = this.props;
+
+    if (!ready) {
+      return <div></div>
+    } 
+    
+    // onboarding
+    if (!hasValidSettings) {
```

Comment:
This is where we redirect for "onboarding" -- setting RPC credentials and wallet creation.

It's kind of janky because user will still see navbar but clicking on navbar items won't have an effect. This is a simpler implementation which is why I did it.

What other options are there?
- Special route without navbar but reusing the `<Settings/>` and `<Create/>` components.
    - Gets weird because redirection behavior after updating settings might be different in onboarding (send to wallet creation) and normal situation (no redirect).
- Modal: modals involve lots of boilerplate
- Maybe pass a flag to `Template` to hide the navbar in some situations?